examples/v1beta1/pipelineruns/output-pipelinerun.yaml

```diff
@@ -1,106 +0,0 @@
-# `PipelineResources` are deprecated, consider using `Tasks` and other replacement features instead
-# https://github.com/tektoncd/pipeline/blob/main/docs/migrating-v1alpha1-to-v1beta1.md#replacing-pipelineresources-with-tasks
-apiVersion: tekton.dev/v1alpha1
-kind: PipelineResource
```

Comment:
In addition to a git resource, this example demonstrates an ability to write to workspace and then read the content written before. Please make sure there is a similar example included as part of our CI before removing it. Or convert this to not include git pipelineresource.

Reply:
Thanks @pritidesai , I think the concern does make much sense. I’ve taken another look at output-pipeline example and I think it mainly targeted to show that how the output resources is being used.  As for the ability to write to the workspace and to use it afterwards I think the [pipelinerun.yaml](https://github.com/tektoncd/pipeline/blob/main/examples/v1beta1/pipelineruns/pipelinerun.yaml#L100-L118) could cover that and the git resolver used here shall correspond to the replacement of our git resources according to the guide https://github.com/tektoncd/pipeline/blob/main/docs/pipelineresources.md.